Modix.Data/Repositories/InfractionRepository.cs

```diff
@@ -104,17 +104,27 @@ public interface IInfractionRepository
         /// </summary>
         /// <param name="infractionId">The <see cref="InfractionEntity.Id"/> value of the infraction to be rescinded.</param>
         /// <param name="rescindedById">The <see cref="UserEntity.Id"/> value of the user that is rescinding the infraction.</param>
-        /// A <see cref="Task"/> which will complete when the operation is complete,
+        /// <returns>A <see cref="Task"/> which will complete when the operation is complete,
         /// containing a flag indicating whether the update was successful (I.E. whether the specified infraction could be found).
         /// </returns>
         Task<bool> TryRescindAsync(long infractionId, ulong rescindedById);
 
+        /// <summary>
+        /// Marks a deleted infraction as active, based on its ID
+        /// </summary>
+        /// <param name="infractionId">The <see cref="InfractionEntity.Id"/> value of the infraction to be restored.</param>
+        /// <param name="restoredById">The <see cref="UserEntity.Id"/> value of the user that is rescinding the infraction.</param>
```

Comment:
Not a major deal, but this does say "rescinding", not that the XML documentation is all too important.

Reply:
Fixed